tests/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ if (ANDROID)
     return()
 endif()
 
+find_package(VulkanLoader CONFIG)
+
```

Comment:
The tests aren't just building, but running in CI. And since its testing a Mock Driver, the Vulkan-Loader is needed to run the tests. Maybe I could figure out directly loading mock icd, but it was simpler to keep it as a test-only dependency. The main purpose of this PR is to prevent anyone wanting to use vkcube/vulkaninfo/mockicd from needing to have the loader build (since it has assembly & some other weird requirements, and isn't supported on android).

Reply:
OK that makes sense. I had thought the VVL team found a way around that using a system package or something, but apparently not: https://github.com/KhronosGroup/Vulkan-ValidationLayers/commit/4c0cde5685020b6b7d44d38fc7e24a1a3d8ef2aa